internal/gengapic/testdata/empty_opt.want

```diff
@@ -10,12 +10,17 @@ func defaultClientOptions() []option.ClientOption {
 		option.WithEndpoint("foo.bar.com:443"),
 		option.WithGRPCDialOption(grpc.WithDisableServiceConfig()),
 		option.WithScopes(DefaultAuthScopes()...),
+		option.WithGRPCDialOption(grpc.WithDefaultCallOptions(
+		grpc.MaxCallRecvMsgSize(-1),
```

Comment:
I don't see the documentation saying that -1 refers to infinity. Did I miss something? If not, maybe we should send a small documentation PR to the gRPC repo?

Reply:
I've sent y'all an email exchange with one of the gRPC TLs where he says this is supported. Furthermore, this is done already in some of the Python GAPICs

Reply:
After looking at the source code, I might be second guessing this behavior for grpc-go specifically. The defaultClientMaxSendMsgSize is `math.maxInt32` ([here](https://github.com/grpc/grpc-go/blob/977142214c45640483838b8672a43c46f89f90cb/clientconn.go#L96)), rather than -1. This also precludes the need for us to set the maxSendMsgSize call option, its already maxed out by default.

Reply:
Just double checked with a grpc-go team member, and they said they don't support -1. I will use the math lib max int value instead. Thanks @pongad for catching this!

Reply:
Done! PTAL